libs/metrics/top_level_metrics.py

```diff
@@ -145,20 +145,12 @@ def calculate_or_copy_test_positivity(
     ) and has_data_in_past_10_days(data[CommonFields.NEGATIVE_TESTS])
     test_positivity = common_df.get_timeseries(data, CommonFields.TEST_POSITIVITY, EMPTY_TS)
     if positive_negative_recent or not test_positivity.notna().any():
```

Comment:
This is probably re-litigating old code, but does this cause us to not use test positivity when calculated in `test_positivity.py` if we have positive tests and negative tests?  I guess we've disabled most inputs of that? 

I guess a large part of this is due to how we choose what goes into postive tests and negative tests in earlier parts of the code.